test/src/test/java/org/corfudb/runtime/checkpoint/CheckpointSmokeTest.java

```diff
@@ -417,4 +421,69 @@ private void writeCheckpointRecords(UUID streamId, String checkpointAuthor, UUID
         }
     }
 
+    @Test
+    public void MultiCheckpointWriter2Test() throws Exception {
+        MultiCheckpointWriterTestInner(true);
+        // While creating & testing MultiCheckpointWriter,
+        // I accidentally discovered that a checkpoint that
+        // immediately follows an earlier checkpoint can cause
+        // map corruption.  Let's test again for posterity.
```

Comment:
for posterity, I actually think that it is better to explain: 
   'test a dense sequence of (two) checkpoints, without any updates in between'